src/megatron/bridge/models/qwen_vl/qwen25_omni_bridge.py

```
@@ -0,0 +1,181 @@
+            text_config.rope_parameters.rope_theta
+            if hasattr(text_config, "rope_parameters") and 
+               text_config.rope_parameters is not None and
+               hasattr(text_config.rope_parameters, "rope_theta")
+            else 1000000.0
+        )
+        
+        model_dtype = self.dtype_from_hf(text_config, default=torch.float32)
+        
+        image_token_id = getattr(thinker_config, "image_token_id", None) or getattr(thinker_config, "image_token_index", 151655)
+        video_token_id = getattr(thinker_config, "video_token_id", None) or getattr(thinker_config, "video_token_index", 151656)
+        audio_token_id = getattr(thinker_config, "audio_token_id", None) or getattr(thinker_config, "audio_token_index", 151646)
```

Flake8 (7.3.0)

[error] local variable 'audio_token_id' is assigned to but never used

(F841)